# PHYS 267 - Assignment 6 Solutions
###### Last Updated: Feb 23, 2022 by Dr. Brenda Lee
***

### Question 1

The average weekly earnings for graduate students without scholarships is $\$$600. Do graduate students with scholarships earn more than $\$$600? A random sample of $n=40$ graduate students with scholarships showed $\overline{x} = \$725$ and $s = \$102$. Test the appropriate hypothesis using a significance level of 0.01.

Total Marks: 9

#### Solution

Hypothesis : Students earn more than $600 with scholarhsips mu>600
Null Hypothesis: Students earn on average $600 with scholarships mu = 600

In [1]:
import numpy as np
from scipy.stats import norm
# all variables given
xbar = 725
s = 102
n = 40
mu = 600 # mu for our null hypothesis

z_score = (xbar- mu)/(s/np.sqrt(n))
pval= norm.sf(z_score)


print("Z score for the following case is", z_score)
print("p-value for the following is", pval)





Z score for the following case is 7.75068053962838
p-value for the sollowing is 4.570068331589579e-15


Since the p-value is way less than 0.01 we reject the null hypothesis, hence people with scholarships would make more than $600/week

### Question 2

Find the appropriate rejection regions for the test statistic $z$ in these cases. You **must use Python code** to present your answers.
1. A left-tailed test at the 1% significance level. [3 marks]
2. A two-tailed test with $\alpha = 0.01$. [3 marks]
3. Suppose that the observed value of the test statistic was $z = -2.41$. For the rejection regions constructed in parts 1 and 2, draw the appropriate conclusion for the tests. [3 marks]

Total Marks: 9

#### Solution

In [4]:
# 1 for a left tailed test statisic should be similar to question 1 but instead of using survival function we would need to use critical z scores
sig = 0.01
z_crit = norm.ppf(0.01)
print("Test score statistics for left-tailed test is", z_crit)
# since z_crit is < than z_score we can reject the Null hypothesis

#2 for two tailed test with α = 0.01, we can follow similar technique but with 1/2 significance on each side 
z_crit2 = norm.ppf(1-sig/2)
print("Test statistic for the 2 tailed case gives us",z_crit2)
# since 


Test score statistics for left-tailed test is -2.3263478740408408
Test statistic for the 2 tailed case gives us 2.5758293035489004


1. All values below below -2.3263 can be rejected
2. All values less than -2.576 and greater than 2.576 can be rejected
3. since -2.41 is less than 2.576 but greater than -2.576 we are not in the rejection region for the 2-tailed case so we fail to reject the null hypothesis
For the left-tailed case -2.41 is less than -2.326 so we can reject the null hypothesis

### Question 3

A few years ago, a study was done to determine what "normal" body temperature is. A random sample of 130 human body temperatures had a mean of 36.81 degree Celcius and a standard deviation of 0.73 degrees. But... physicians usually say that the average temperature is 37 degrees. Does this new data mean that the average body temperature for a healthy person is different than 37 degrees? Test using the following methods below.
1. Use the critical value approach (ie. finding critical z-score and test statistic z-score, and comparing them within the rejection region) and $\alpha = 0.05$ [5 marks]
2. Use the p-value approach (ie. finding the test statistic z-scoore and its corresponding p-value to be less than $\alpha$ [4 marks]
3. Compare the conclusions from Part 1 and Part 2. Are they the same, or different? Explain. [1 mark]
4. Let's say that the 37 degrees norm was drawn from a doctor in 1868, who claimed to have recorded 1 million temperatures during his research. What conclusions can you draw about his research in light of your conclusions from the above questions? [2 marks]

Total Marks: 12 marks

#### Solution

In [7]:
# Part 1
# H0 or null Hypothesis is mu = 37
# H1 or alternate hypothesis is mu ≠ 37
# constants and vlaues given
mu = 37
n2 = 130
xbar2 = 36.81
std2 = 0.73
alpha = 0.05
# z-score
z2 = (xbar2-mu)/(std2/np.sqrt(n2))
print("Z score for this case is", z2)

# z crit
# two tailed case, this works 
z_crit3 = norm.ppf(1-alpha/2)
print("Z crit for the following is", z_crit3)



Z score for this case is -2.967579873545666
Z crit for the following is 1.959963984540054


since z score is outside the acceptance range hence we would need to reject the null hypothesis

In [8]:
# Part 2
# Z score remains the same, so we can proceeed with p-value test
p_value = 2*norm.cdf(z2)
print("the p-value is ",p_value)

the p-value is  0.0030015428506652013


since the p-value is less than 0.05 we can reject the null hypothesis there by concluding that temperature is different from 37 similar to the z crit test case

# Part 3
Both conclusions are the same we reject Null hypothesis. For zcrit, the z score lies in the rejection region of the two tailed test. And for the p-value case since it does not exceed the siginificance of 0.05 we can reject null hypothesis and accept alternate hypothesis

# Part 4
The mean value of 37 is definitely possible. The exact method of acquiring this data, number of trials and true population size is unknown. Moreover, we don't know about environmental or other factors. In fact in recent studies it has been found that the body temperature has been decreasing by 0.03 C per decade. https://elifesciences.org/articles/49555 